specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1784,14 +1819,12 @@ In een stuk is daarom óf een kadasterstuk óf een terInschrijvingAangebodenStuk
           items:
             $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink"
         zakelijkGerechtigden:
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink"
-#        nietBagAdressen:
-#          title: "betreft"
-#          type: "array"            Is geen endpoint voor Komt te vervallen ?
-#          description: ""
-#          minItems: 1
-#          items:
-#            $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/schemas/HalLink"
+          title: ""
+          type: "array"
+          description: ""
+          minItems: 1
```

Comment:
minItems moet eruit, anders is gebruik fields hierop niet meer toegestaan.

Reply:
Voer ik door. Ook voor de andere relaties waarbij minitems: 1 is opgenomen. (bagAdressen en stukken)